.gitignore

```diff
@@ -3,9 +3,9 @@ logs/*
 node_modules/
 bower_components/
 tmp
+!dist/index.html
```

Comment:
Why do you need this rule? `index.html` can be placed in the root folder. No need to modify it.

Reply:
I just use this universal .gitignore file for different projects))
The rule was removed.